### DISCUSSION TITLE
Evasive action (GENERAL) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#21] [Card Balance]
Hidden in the shadow
Play on a Warrior at any time. The Warrior gains +3 to [A] until the beginning of your next Draw step.

Evasive action
Play during COMBAT. Your WARRIOR gains +3 to [A] until end of COMBAT
:arrow_right: perhaps a bonus to +4?

Fall Prone
Play on your combatant. The Warrior goes into Cover (even if it is the Attacker) and gains +5 to [A] instead of +3. After COMBAT, the Warrior remains in Cover, but at +3 to [A] .
:skull: Shouldn't be play on "Bio-giant" or "eradicator deathdroid". :skull:

**Comment by Sol-au-fein** [comment]
I was thinking Hidden in Shadows could give some small penalty to F/S in exchange of longer boost.

**Reply by Kite80** [reply]
What if a warrior hidden in the shadows can't retaliate during combat? You can't stay hidden in the shadows that much if you shoot with your gun or try to hit someone with a blade after all...

**Reply by Sol-au-fein** [reply]
Yeah, i like your idea!

**Reply by Sol-au-fein** [reply]
Another suggestion is disable just shooting. Because of muzzle flash reveal his position.

**Comment by 21-ric** [comment]
"Evasive action" +3 to [A]. 
Your opponent's cards can't prevent this card.
In my idea, you can play this card against "Free marine" and if your opponent played "cornered"
Your opponent can't cancel this card.

What do you think?

**Reply by Sol-au-fein** [reply]
No fate cards means No fate cards

**Reply by 21-ric** [reply]
Excected if a card says the opposite. 🤡
